operator/internal/controller/odigos_controller.go

```diff
@@ -62,6 +63,32 @@ type OdigosReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+// relatedImageEnvVars is a reverse-lookup map to quickly find the environment variable
+// for component images when openshift is enabled
+var relatedImageEnvVars = map[string]string{
+	k8sconsts.AutoScalerImageName: "RELATED_IMAGE_AUTOSCALER",
+	k8sconsts.AutoScalerImageUBI9: "RELATED_IMAGE_AUTOSCALER",
+
+	k8sconsts.OdigosClusterCollectorImage:     "RELATED_IMAGE_COLLECTOR",
+	k8sconsts.OdigosClusterCollectorImageUBI9: "RELATED_IMAGE_COLLECTOR",
+
+	k8sconsts.InstrumentorImage:               "RELATED_IMAGE_INSTRUMENTOR",
+	k8sconsts.InstrumentorImageUBI9:           "RELATED_IMAGE_INSTRUMENTOR",
+	k8sconsts.InstrumentorEnterpriseImage:     "RELATED_IMAGE_ENTERPRISE_INSTRUMENTOR",
+	k8sconsts.InstrumentorEnterpriseImageUBI9: "RELATED_IMAGE_ENTERPRISE_INSTRUMENTOR",
+
+	k8sconsts.UIImage:     "RELATED_IMAGE_FRONTEND",
+	k8sconsts.UIImageUBI9: "RELATED_IMAGE_FRONTEND",
+
+	k8sconsts.OdigletImageName:           "RELATED_IMAGE_ODIGLET",
+	k8sconsts.OdigletImageUBI9:           "RELATED_IMAGE_ODIGLET",
+	k8sconsts.OdigletEnterpriseImageName: "RELATED_IMAGE_ODIGLET",
+	k8sconsts.OdigletEnterpriseImageUBI9: "RELATED_IMAGE_ODIGLET",
```

Comment:
not sure if that's a typo or not (since I saw that these ENTERPRISE envs are defined)

```suggestion
	k8sconsts.OdigletEnterpriseImageName: "RELATED_IMAGE_ENTERPRISE_ODIGLET",
	k8sconsts.OdigletEnterpriseImageUBI9: "RELATED_IMAGE_ENTERPRISE_ODIGLET",
```

Reply:
great catch thank you